### ISSUE TITLE
Small help section at top of edit page is occluded

### ISSUE THREAD

**Issue by fwolfst** [cosmocode/dokuwiki-template-sprintdoc#23]
In my instance, the small "help edit section" ("Seite Bearbeiten und [Speichern] drücken. Siehe syntax zur Wiki-Syntax. Bitte nur editieren, [...]"), is occluded by the gray bar just above it. In contrast to e.g.  https://demo.sprintdoc.de/start?do=edit where it works as it should.

I'll gladly provide further info or screenshots of it. Running latest template and dokuwiki versions.

**Comment by splitbrain** [comment]
Screenshot please

**Comment by fwolfst** [comment]
![wiki_header_edit_help](https://user-images.githubusercontent.com/219373/51592156-382cfc00-1eef-11e9-870e-c3490351b916.jpg)

**Comment by fwolfst** [comment]
Also, setting a logo and removing all user-css (had made two tiny changes for another template) did not change the look.

**Comment by micgro42** [comment]
I can "reproduce" something like this if I manually remove the message area. But that is part of the main template and intended to be always there. So maybe a plugin doing something strange?

See:
https://github.com/cosmocode/dokuwiki-template-sprintdoc/blob/645b04c3411c9ae7396a76d2570ab6d2417265a2/main.php#L349

**Comment by fwolfst** [comment]
Thanks. Indeed if I deactivate FastWiki (https://github.com/zioth/dokuwiki_fastwiki) it works. I will file an issue on that project, as I cannot judge on which side to deal with the issue best.

**Comment by zioth** [comment]
This might be fixable using the `fastwiki` hooks in your template:
https://www.dokuwiki.org/plugin:fastwiki:integration

Or, if I have time at some point, I'll either submit a pull request, or make a special case for `sprintdoc` in `fastwiki`.

**Comment by splitbrain** [comment]
PR would be welcome

**Comment by zioth** [comment]
Adding this to your CSS will fix the issue:
```
/* Fastwiki support */
.mode_edit .content .row > .col-xs-12 #dokuwiki__content::before {
	display: none
}
```

That will hide the unused TOC bar in edit mode. Didn't seem worth the effort of creating a fork. If you'd prefer a PR, I can do it. :)

**Comment by annda** [comment]
Great, the style is included in b05c3b3ce4c1030e25af78c3fe7962f264e668cd

**Comment by zioth** [comment]
thanks!
